test/WebJobs.Script.Tests.Integration/Controllers/WarmupScenarios.cs

```diff
@@ -87,6 +86,11 @@ public async Task TestWarmupEndPoint_WhenHostStarts()
                 $"Instead found {response.StatusCode}");
         }
 
+        public void Dispose()
+        {
+            _testHost.Dispose();
```

Comment:
make this `_testHost?.Dispose();`, just in case something weird happens and testhost isn't created yet.